assets/src/service/client.js

```diff
@@ -1,7 +1,9 @@
-import ApolloClient from 'apollo-boost'
+import { ApolloClient, InMemoryCache } from '@apollo/client'
 import Cookie from 'js-cookie'
 
 export default new ApolloClient({
+  uri: '/graphql',
+  cache: new InMemoryCache(),
```

Comment:
What is the advantage of adding this?

Reply:
It seemed to be required with the change in Apollo versions.